apps/fabric-website/README.md

```diff
@@ -9,51 +9,18 @@ Office UI Fabric is a collection of projects that represent the Office and Offic
 Before you get started, make sure you have [node.js](https://nodejs.org/), [gulp](http://gulpjs.com/), and [git](https://git-scm.com/) installed. To view the documentation including examples, contracts, current statuses, and to add functionality or fix issues locally, you can:
 
 1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git` - this is the [Office UI Fabric React project](https://github.com/OfficeDev/office-ui-fabric-react) which you will want to name *office-ui-fabric-react*
-2. `npm install` the Fabric React project you just cloned in the previous step
-3. `gulp serve` the Fabric React project
-4. Stop the local web server
-5. `git clone https://onedrive.visualstudio.com/DefaultCollection/Design/_git/fabric-website` (the Fabric website project) into a sibling directory to the *office-ui-fabric-react* folder you just created and cloned Fabric React into
-6. `npm install` from within the Fabric website folder you created in the previous step
-7. `npm link ../office-ui-fabric-react` This tells NPM to use your local build of Fabric React instead of the regular copy where the link is to the project you cloned in step **1**
-8. `gulp serve`
-
-This will open a web browser with the home page. You can make changes to the code which will automatically build and refresh the page using live-reload.
-
-## Deploy the website
```

Comment:
This doesn't work anymore and will be updated once we have a smooth process going.